shared/SpectralAudioPluginUi.cpp

```diff
@@ -71,6 +71,7 @@ SpectralAudioPluginUi::SpectralAudioPluginUi(
 
 SpectralAudioPluginUi::~SpectralAudioPluginUi()
 {
+    parameterContainer = nullptr;
```

Comment:
I wonder if, in the future, it would be better to access this via the parameterViewPort instead so there is never any confusion about ownership.

However, nulling ptr here makes it clear

Reply:
In fact,I forgot to delete this code when using smart pointer.But it doesn't matter.
This code is unnecessary.may be i should remove it?

Reply:
Ok let's remove it since it's not a smart pointer anymore 👍  Then we can merge in this PR 🥳